blocks/library/list/index.js

```diff
@@ -15,6 +15,42 @@ import BlockControls from '../../block-controls';
 
 const { children, prop } = hpq;
 
+const fromBrDelimitedContent = function( content ) {
+	const listItems = [];
+	listItems.push( createElement( 'li', [], [] ) );
+	content.forEach( function( element, elementIndex, elements ) {
```

Comment:
Minor Nitpick: I think we might want to use arrow function for these callbacks to keep consistent.